log-manager/src/main/java/io/airlift/log/Level.java

```diff
@@ -0,0 +1,77 @@
+package io.airlift.log;
+
+import java.util.Locale;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+enum Level
+{
+    OFF {
+        @Override
+        protected java.util.logging.Level toJulLevel()
+        {
+            return java.util.logging.Level.OFF;
+        }
+    },
+    DEBUG {
+        @Override
+        protected java.util.logging.Level toJulLevel()
+        {
+            return java.util.logging.Level.FINE;
+        }
+    },
+    INFO {
+        @Override
+        protected java.util.logging.Level toJulLevel()
+        {
+            return java.util.logging.Level.INFO;
+        }
+    },
+    WARN {
+        @Override
+        protected java.util.logging.Level toJulLevel()
+        {
+            return java.util.logging.Level.WARNING;
+        }
+    },
+    ERROR {
+        @Override
+        protected java.util.logging.Level toJulLevel()
+        {
+            return java.util.logging.Level.SEVERE;
+        }
+    };
+
+    abstract java.util.logging.Level toJulLevel();
+
+    static Level fromJulLevel(java.util.logging.Level level)
+    {
+        // Convert any implementation of Level based on the int value
+        int levelValue = level.intValue();
+        if (levelValue == java.util.logging.Level.OFF.intValue()) {
+            return OFF;
+        }
+        if (levelValue >= java.util.logging.Level.SEVERE.intValue()) {
+            return ERROR;
+        }
+        if (levelValue >= java.util.logging.Level.WARNING.intValue()) {
+            return WARN;
+        }
+        if (levelValue >= java.util.logging.Level.INFO.intValue()) {
+            return INFO;
+        }
+        return DEBUG;
+    }
+
+    static Level parse(String name)
+    {
+        checkNotNull(name, "name is null");
+        name = name.toUpperCase(Locale.US);
+        for (Level level : values()) {
```

Comment:
This is the same behavior of the logback `Level.toLevel()` method that we currently use when processing the log levels file. I was simply suggesting a simpler implementation.